pkg/sysregistriesv2/testdata/aliases.conf

```diff
@@ -0,0 +1,4 @@
+[aliases]
+fooDocker="docker.io/library/foo:latest"
```

Comment:
Hum, do we allow upper-case in aliases? In a sense, it would be great to have that (in the extreme _requiring_ upper-case would even make it unambiguous that the alias is not a `docker.io`/search short-name).

Either way, we need to explicitly document the allowed syntax. Maybe `regexp.go:nameComponentRegexp`, possibly with upper case allowed as well?

---

I haven’t checked but I’m somewhat sure that Kubernetes would be unhappy with non-`reference.Named` input (i.e. upper-case)

But, on second thought, It seems attractive to say that should explicitly _not_ be supported by CRI-O, even if Podman on the same host supports them , and even if CRI-O respects the short-name setting to configure what it does on search (reject short names completely / use old search behavior). Users of k8s don’t _need_ aliases as much, most of the configuration is somehow software-generated in either case (I _guess_, with no data), so let’s not introduce that concept for all of k8s to deal with?

OTOH consistency between Podman and CRI-O _is_ valuable. I don’t know.

Reply:
I leave this open as I don't yet have a feeling for a potential syntax.

Reply:
I think stick to lowercase, so we don't have to deal with Fedora being different then fedora.

Reply:
Still unadressed in the PR.

Reply:
The PR’s validation currently disables upper-case because it uses `reference.Parse*`.